src/form.js

```diff
@@ -98,14 +98,12 @@ export default React.createClass({
     propTypes: {
```

Comment:
I believe react does some voodoo when calling functions that don't exist to not make things crash. But yes, we should probably add some defaults :smile: